spec/rspec/core/backtrace_formatter_spec.rb

```diff
@@ -112,19 +112,7 @@ def make_backtrace_formatter(exclusion_patterns=nil, inclusion_patterns=nil)
 
         expect(BacktraceFormatter.new.format_backtrace(backtrace)).to eq(["./my_spec.rb:5"])
       end
-
-      it "excludes lines from rspec libs by default", :failing_on_appveyor, :if => RSpec::Support::OS.windows? do
-        backtrace = [
-          "\\path\\to\\rspec-expectations\\lib\\rspec\\expectations\\foo.rb:37",
-          "\\path\\to\\rspec-expectations\\lib\\rspec\\matchers\\foo.rb:37",
-          ".\\my_spec.rb:5",
-          "\\path\\to\\rspec-mocks\\lib\\rspec\\mocks\\foo.rb:37",
-          "\\path\\to\\rspec-core\\lib\\rspec\\core\\foo.rb:37"
-        ]
-
-        expect(BacktraceFormatter.new.format_backtrace(backtrace)).to eq([".\\my_spec.rb:5"])
-      end
-
+      
```

Comment:
> Sure, shall I squash them into a single commit and place it at the head of history?

Please do squash into a small set of logical commits.  It need not be one -- use your judgement for what tells the most clear story for when one of us (likely me) looks through these commits again in the future when I'm digging through the git history to understand something.

I'm not totally sure what you mean by "place at the head of the history". After you squash, you can force push to this branch...is that what you meant?

> (also, what happens to these conversations if I squash? I dislike the idea of losing the context and considerations that explain the code. To the point, even, of wishing they became part of the git history.)

I'm a serial rebaser -- I really want my git history to tell a clear story, so it's the standard way I reconcile a feature branch with changes from master as long as I'm working with a team that treats feature branches as "owned" by the originator unless otherwise stated (meaning I can assume no one is working off of the branch).  Having rebased and force pushed to PR branches hundreds of times on GitHub, I've never noticed any comments ever getting lost.  I'm not 100% how they do it but the comments remain in my experience.